client/me/purchases/remove-purchase/remove-domain-dialog/index.jsx

```diff
@@ -67,7 +76,7 @@ class RemoveDomainDialog extends Component {
 				{ this.renderDomainDeletionWarning( productName ) }
 
 				<p>
-					{ translate( 'If you want to use this domain with another service, DO NOT delete it.' ) }{ ' ' }
+					{ translate( 'If you want to use this domain with another service, do not delete it.' ) }{ ' ' }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.